zgw/documenten-api/src/main/kotlin/com/ritense/portal/documentenapi/autoconfigure/DocumentenApiAutoConfiguration.kt

```diff
@@ -40,11 +46,34 @@ class DocumentenApiAutoConfiguration {
         return DocumentenApiService(documentenApiClient, documentenApiConfig)
     }
 
+    @Bean
+    @ConditionalOnMissingBean(VirusScanService::class)
+    fun virusScanService(
+        clamAVClient: ClamavClient
+    ): VirusScanService {
+        return ClamAVService(clamAVClient)
+    }
+
+    @Bean
+    fun clamAVClient(
+        clamAVVirusScanConfig: ClamAVVirusScanConfig
```

Comment:
De ClamAV config zou zijn eigen autoconfiguration moeten gebruiken met een `@ConditionalOnProperty`. Anders kunnen we nooit een andere implementatie gebruiken zonder ook ClamAV te moeten configureren.

Reply:
Kan je precies uitleggen wat je hiermee bedoeld?? Op welke Bean zouden we de @ConditionalOnProperty het beste kunnen zetten??

Reply:
Volgens mij kunnen we nog steeds geen implementatie maken zonder ClamAV, of zelfs zonder virusscanner. Deze feature zou volledig optioneel moeten zijn. Graag alle ClamAV beans in een eigen AutoConfiguration plaatsen met een `ConditionalOnProperty`